build.gradle

```diff
@@ -231,7 +231,7 @@ subprojects {
     jvmArgs += project.property('extraJvmArgs')
 
     testLogging {
-      events "failed"
+      events "started", "passed", "skipped", "failed"
```

Comment:
I remember that #1210 added more output logging, which was immediately reverted by #1212, so we might be a little cautious with having too much output